tester/lib/cli.js

```diff
@@ -155,35 +154,33 @@ export async function main(argv = process.argv.slice(2)) {
   };
   const wasmImportsPath = path.join(cwd, imports);
 
-
-if (config['--imports'] == pathAstester){
-  let modWasmImport;
-  modWasmImport = await import('file://' + wasmImportsPath);
-  const modExport = modWasmImport.local(memory);
-
-  const wasmImports = await asyncfs.access(wasmImportsPath)
-    .then(async () => {
-      return Object.assign(modExport, wasiImports);
-    })
-    .catch(() => wasiImports);
-
-  const instance = await WebAssembly.instantiate(mod, wasmImports);
-  wasi.initialize(instance);
-  modWasmImport.setExports(instance.exports);
-  instance.exports._startTests();
-}
-else
-{
-  const wasmImports = await asyncfs.access(wasmImportsPath)
-    .then(async () => {
-      const modWasmImport = await import('file://' + wasmImportsPath);
-      return Object.assign(modWasmImport.default(memory), wasiImports);
-
-    })
-    .catch(() => wasiImports);
-
-  const instance = await WebAssembly.instantiate(mod, wasmImports);
-  wasi.initialize(instance);
-  instance.exports._startTests();
-
+  if (config['--imports'] == pathAstester) {
+    let modWasmImport;
+    modWasmImport = await import('file://' + wasmImportsPath);
+    const modExport = modWasmImport.local(memory);
+
+    const wasmImports = await asyncfs.access(wasmImportsPath)
+        .then(async () => {
+          return Object.assign(modExport, wasiImports);
+        })
+        .catch(() => wasiImports);
+
+    const instance = await WebAssembly.instantiate(mod, wasmImports);
+    wasi.initialize(instance);
+    modWasmImport.setExports(instance.exports);
+
+    process.exitCode = instance.exports._startTests();
+  } else {
+    const wasmImports = await asyncfs.access(wasmImportsPath)
+        .then(async () => {
+          const modWasmImport = await import('file://' + wasmImportsPath);
+          return Object.assign(modWasmImport.default(memory), wasiImports);
+        })
+        .catch(() => wasiImports);
```

Comment:
same here why adding 2 spaces? when I run `npm run fmt` on main branch it modify lot of files and exit with an error, we should use as-prettier, do you want me to create an issue for this?

Reply:
yep please :-) And share with me config pretty please

Reply:
I've create #27